TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/ExpressionBasedNodeLabelGenerator.php

```diff
@@ -71,13 +74,27 @@ public function initializeObject()
      */
     public function getLabel(NodeInterface $node, $crop = true)
     {
-        $label = \TYPO3\Eel\Utility::evaluateEelExpression($this->getExpression(), $this->eelEvaluator, array('node' => $node), $this->defaultContextConfiguration);
+        $label = null;
+        if ($node->getNodeType()->isOfType('TYPO3.Neos:ContentCollection') && $node->isAutoCreated()) {
```

Comment:
I agree, but why does it even need to only be for content collections? isn't the fact that it's auto-created and has the label property set enough?